src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 14px;
   margin: 0;
+
+  --hover-color: #00acdc;
+}
+
+a {
+  color: #000;
+  text-decoration: none;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+nav ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+}
+
+nav li {
+  margin-left: 20px;
+}
+
+nav li:first-child {
+  margin: 0;
+}
+
+nav a {
+  display: flex;
+  text-transform: uppercase;
+  padding: 23px 0;
+  position: relative;
+}
+
+nav a:hover,
+.is-active {
+  color: var(--hover-color);
```

Comment:
The hover effect and active state use the CSS variable `--hover-color`. Ensure this variable is defined in a way that it provides sufficient contrast against the background for accessibility.